### ISSUE TITLE
NTR: Bone marrow: monocytes increased (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9177]
New term request Bone marrow: monocytes increased (elthat_100822133511):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000641
- label: Bone marrow monocytosis
- parents: [Abnormal bone marrow cell morphology (HP:0005561)](https://hpo.jax.org/app/browse/term/HP:0005561)
- pmid: 
- def: Increased count of monocytes in the bone marrow.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9177
- synonyms: Bone marrow monocytes increased(NONE)
